src/v/raft/tests/raft_fixture.h

```diff
@@ -144,6 +144,10 @@ class in_memory_test_protocol : public consensus_client_protocol::impl {
     prefix_logger& _logger;
 };
 
+inline model::timeout_clock::time_point default_timeout() {
+    return model::timeout_clock::now() + 30;
```

Comment:
is it really seconds by default?I would have expected 30s or similar

Reply:
right, i missed this